packages/morph-attr/lib/main.js

```diff
@@ -110,7 +110,7 @@ AttrMorph.prototype.setContent = function (value) {
   if (this.lastValue === value) { return; }
   this.lastValue = value;
 
-  if (this.escaped) {
+  if (this.escaped && this.valueIdentity(value)) {
```

Comment:
ah, this aims to skips escaping if the value is falsy. As we believe no falsey values will need escaping.